src/screens/HomeScreen.js

```diff
@@ -99,11 +99,96 @@ export const HomeScreen = ({ navigation, route }) => {
     }, 500);
   };
 
+  const flatListData = [
+    {
+      categoriesNews,
+      fetchPolicy,
+      isShow: showNews,
```

Comment:
i think "is show" is weird to read and understand. what about `showData`?

Reply:
Yes, it'll be more understandable.